components/signup/signup_controller/index.js

```diff
@@ -17,8 +18,9 @@ import SignupController from './signup_controller.jsx';
 function mapStateToProps(state, ownProps) {
     const license = getLicense(state);
     const config = getConfig(state);
-
+    const subscriptionStats = subscriptionStatsFromState(state);
     const isLicensed = license && license.IsLicensed === 'true';
+    const isCloud = license && license.Cloud === 'true';
```

Comment:
Should we just have the subscriptionsStats selector return whether its cloud?

Reply:
Hi @nickmisasi , that makes sense, I will add that validation.